cciptestinterfaces/interface.go

```diff
@@ -25,23 +25,75 @@ for CCIP16 and CCIP17
 // it deploys network-specific infrastructure, configures both CL nodes and contracts and returns
 // operations for testing and SLA/Metrics assertions.
 type CCIP17ProductConfiguration interface {
-	Testable
+	Chains
 	Observable
 	OnChainConfigurable
 	OffChainConfigurable
 }
 
 // Observable pushes Loki streams and exposes Prometheus metrics and returns queries to assert SLAs.
 type Observable interface {
-	ExposeMetrics(ctx context.Context, addresses, chainIDs, wsURLs []string) ([]string, *prometheus.Registry, error)
+	ExposeMetrics(ctx context.Context, source, dest uint64, chainIDs, wsURLs []string) ([]string, *prometheus.Registry, error)
 }
 
-// Testable provides functions for a standardized CCIP test suite.
-type Testable interface {
-	// SendArgsV2Message sends a message with ArgsV2 params
-	SendArgsV2Message(ctx context.Context, e *deployment.Environment, addresses []string, src, dest uint64) error
-	// SendArgsV3Message sends a message with ArgsV3 params
-	SendArgsV3Message(ctx context.Context, e *deployment.Environment, addresses []string, selectors []uint64, src, dest uint64, finality uint16, execAddr, receiverAddr string, execArgs, tokenArgs []byte, ccv, optCcv []protocol.CCV, threshold uint8) error
+// TokenAmount represents a token amount being sent in a CCIP message.
+type TokenAmount struct {
```

Comment:
After adding these types implementing the interface in ccv-evm was causing circular imports. We could potentially move this to `protocol` if it makes sense to authors.